sql/2-seed.sql

```diff
@@ -23,13 +23,13 @@ INSERT INTO crew_permission (name) VALUES ("MakeTask");
 INSERT INTO crew_permission (name) VALUES ("EditTask"); 
 INSERT INTO crew_permission (name) VALUES ("DeleteTask"); 
 
-INSERT INTO permission (name, id) VALUES ("CrudRole", 5); 
-INSERT INTO permission (name, id) VALUES ("SetUserRole", 6); 
-INSERT INTO permission (name, id) VALUES ("UpdateUserRole", 7); 
-INSERT INTO permission (name, id) VALUES ("CrudPermission", 8); 
-INSERT INTO permission (name, id) VALUES ("AddPermissionToRole", 9); 
-INSERT INTO permission (name, id) VALUES ("ViewUserInformation", 10); 
-INSERT INTO permission (name, id) VALUES ("DeleteUserAccount", 11); 
+INSERT INTO permission (name, id) VALUES ("CrudRole"); 
+INSERT INTO permission (name, id) VALUES ("SetUserRole"); 
+INSERT INTO permission (name, id) VALUES ("UpdateUserRole"); 
+INSERT INTO permission (name, id) VALUES ("CrudPermission"); 
+INSERT INTO permission (name, id) VALUES ("AddPermissionToRole"); 
+INSERT INTO permission (name, id) VALUES ("ViewUserInformation"); 
+INSERT INTO permission (name, id) VALUES ("DeleteUserAccount"); 
```

Comment:
id ?

Reply:
I fixed it, but i dont know why it is still there